### ISSUE TITLE
Vscode reading issue with screenreaders

### ISSUE THREAD

**Issue by snufas** [microsoft/vscode#79317]
- Version: 1.37.1 (user setup)
-  Commit: f06011ac164ae4dc8e753a3fe7f9549844d15e35
- Date: 2019-08-15T16:17:55.855Z
- Electron: 4.2.7
- Chrome: 69.0.3497.128
- Node.js: 10.11.0
- V8: 6.9.427.31-electron.0
- OS: Windows_NT x64 10.0.17763

Steps to Reproduce:

hello, I have a problem with nvda reading code in the  visual studio code application. I am  providing example code below and how nvda reads the code I took the output from the speech viewer.
the code sample I am trying to read 

text = print("hello world ")
vs code speech reading by word using ctrl plus right arrow. 

- text = 
- text = 
- print("
- hello 
- hello 
- world ")
- world ")

so, from the output above we can clearly  identify that it is not correct simply because the information is repeated which confuses me where my cursor is. 
notepad++  same code output 

- =
- print
- ("
- hello
- world
- ")

for me notepad++  works much better then vs code does. I know it is related to electron being used in vs code, so the question is: is  something possible to do to solve the problem either by changing something in nvda or vs code? I just want to work with vs code Reliably  since it is so powerful, and everyone seams to use the editor now a days plus there is a tun of extensions available. 
For me the work around this problem is use ctrl plus shift plus right arrow, so I am selecting the code an then it reads the text as it should without  repetition. 
I wonder on which end the problem exists? I suspect that it is a nvda problem since cursor moves saime way,  on both notepad++ and vs code.  I am Shure that I am not the only one who is encountering  this problem, so there for it would be nice if someone could investigate this issue, so developers please notice this issue because it is important… 

I am posting same issue on NVDA and vs code github pages so it may get attention from both sides. 
Thanks for reading and I am hoping that this issue could be addressed  and fixed in the  future.

**Comment by isidorn** [comment]
@snufas thanks a lot for posting this issue on GitHub and for providing feedback.
This seems like a duplicate of https://github.com/microsoft/vscode/issues/28306
Can you please confirm, if yes it would be great if we could continue the discussion there. Other users have also complained about that one and we plan to adress it in the future.
Thanks!

**Comment by snufas** [comment]
@isidorn  
no it is not duplicate  i posted saime issue a while back in nvda github page 
https://github.com/nvaccess/nvda/issues/7605
I think it is a good idea to leve this open because i explained this quite whell i think.  
Thanks for responding.

**Comment by snufas** [comment]
this is saime issue on nvda github page that i posted here.
https://github.com/nvaccess/nvda/issues/10105

**Comment by Neurrone** [comment]
This is a duplicate of #28306, which has a lot more technical details. I suggest closing this and continuing the discussion there.

**Comment by isidorn** [comment]
@snufas I can leave this open, however it is almost certanly a duplicate. Thus I recommend that you follow the discussion in #28306 and you can also provide feedback there.
Thus I am closing this, and when we fix #28306 if this does not get fixed we can reopen the issue.
Thanks for understanding.

**Comment by snufas** [comment]
@isidorn it is ok with me how ever  that issue is inactive. May you could ad relative people to join the discussion?   I think the devs should be aware of this issue, so it get fixed sooner rather then later.
Thanks for referencing the issue.

**Comment by isidorn** [comment]
@snufas I am one of the devs and we have that issue on our radar. We plan to look into it in the near future. Thanks!
